TileFormats/FeatureTable/README.md

```diff
@@ -0,0 +1,71 @@
+
+## Layout
+
+The Feature Table is composed of two parts: a JSON header and an optional binary body. The JSON keys are tile-format-specific semantics, and the values can either be defined directly in the JSON, or refer to sections in the binary body.
+The binary body is a tightly packed binary buffer containing data used by the header. It is more efficient to store long numeric arrays in the binary body.
+
+**Figure 1**: Feature Table layout
+
+![feature table layout](figures/feature-table-layout.png)
+
+Code for reading the Feature Table can be found in [Cesium3DTileFeatureTableResources.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Cesium3DTileFeatureTableResources.js) in the Cesium implementation of 3D Tiles.
+
+## JSON Header
+
+Feature table values can be defined in the JSON header in three different ways.
+
+1. A single JSON value. (e.g. `"INSTANCES_LENGTH" : 4`)
+  * This is common for global semantics like `"INSTANCES_LENGTH"` and `"POINTS_LENGTH"`.
+2. A JSON array of values. (e.g. `"POSITION" : [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]`)
+    * Feature values are always stored as a single, flat array, not an array of arrays.  Above, each `POSITION` refers to a `float32[3]` data type so there are three features: `Feature 0's position`=`(1.0, 0.0, 0.0)`, `Feature 1's position`=`(0.0, 1.0, 0.0)`, `Feature 2's position`=`(0.0, 0.0, 1.0)`.
+3. A reference to data in the binary body, denoated by an object with a `byteOffset` property. (e.g. `"SCALE" : { "byteOffset" : 24` } )
+  * `byteOffset` is always relative to the start of the binary body.
+
+The only valid keys in the JSON header are the defined semantics by the tile format.  Application-specific data should be stored in the Batch Table.
+
+## Binary Body
+
+When the JSON header includes a reference to the binary, the provided `byteOffset` is used to index into the data. 
+
+**Figure 2**: Indexing into the Feature Table binary
+
+![feature table binary index](figures/feature-table-binary-index.png)
+
+Values can be retrieved using the number of features, `featuresLength`, the desired feature id, `featureId`, and the data type for the feature semantic.
+
+For example, using the `POSITION` semantic, which has a `float32[3]` data type:
+
+```javascript
+var byteOffset = featureTableJSON.POSTION.byteOffset;
+
+var positionArray = new Float32Array(featureTableBinary.buffer, byteOffset, featuresLength * 3); // There are three components for each POSITION feature.
+var position = positionArray.subarray(featureId * 3, featureId * 3 + 3); // Using subarray creates a view into the array, and not a new array.
+```
+
+## Implementation Notes
+
+In JavaScript, a `TypedArray` cannot be created on data unless it is byte-aligned to the data type.
```

Comment:
Padding at the start of the binary body is not easily possible. When you add padding to the binary the `byteOffset` values in the JSON need to change, which then may extend the JSON length and breaks the alignment of the binary.

I believe this is the same reason why binary gltf appends whitespace to the JSON rather than padding the binary.